api/src/assets/test_assets.py

```diff
@@ -42,6 +42,19 @@ def test_create_asset(auth_client):
     for attr in new_asset:
         assert new_asset[attr] == queried_asset.__dict__[attr]
 
+# # Currently will fail, since the api does not check for invalid location
```

Comment:
Rather than comment out a test, use the `skip` decorator from pytest.
See https://docs.pytest.org/en/latest/skipping.html

Reply:
Will be fixed!